packages/vite-plugin-shopify/src/html.ts

```diff
@@ -22,13 +21,6 @@ export default function shopifyHTML (options: ResolvedVitePluginShopifyOptions):
     configResolved (resolvedConfig) {
       // Store reference to resolved config
       config = resolvedConfig
-
-      // Check for alias generated by vite-plugin-shopify-modules
-      const modulesAlias = config.resolve.alias.find((value) => value.find === '@modules')
-      if (modulesAlias != null) {
-        // Store relative path to modules directory
-        modulesPath = path.relative(options.entrypointsDir, modulesAlias.replacement)
```

Comment:
This code was necessary to support the "shorthand" imports from the modules plugin. This is different from aliases, since it expands `modules/module-name` to `modules/module-name/module-name.js`. I think this is a useful feature to keep, and will have no effect if the modules plugin is not in use.